dev/plugins/hu.elte.txtuml.api.stdlib/src/hu/elte/txtuml/api/stdlib/timers/Timer.xtxtuml

```diff
@@ -28,18 +35,38 @@
 	 *            the signal which is to be sent after the delay
 	 * @param millisecs
 	 *            the time in millisecs to wait before sending the signal
-	 * @return a timer instance to manage this delayed send operation before it
+	 * @return a handle object to manage this delayed send operation before it
 	 *         happens
 	 */
-	static Timer start(ModelClass targetObj, Signal signal, int millisecs) {
-		return new TimerImpl(targetObj, signal, millisecs);
+	public static external-body Timer start(Signal sig, ModelClass targetObj, int millisecs) {
```

Comment:
Is `external-body` necessary here? The called `create` seems to be an `external-body` function as well. The same goes for [`add`](https://github.com/ELTE-Soft/txtUML/blob/5ea692b4cb20a19029876b3edccc1db6d916d78e/dev/plugins/hu.elte.txtuml.api.stdlib/src/hu/elte/txtuml/api/stdlib/timers/Timer.xtxtuml#L87-L99).